userPrefs.jsonc

```diff
@@ -34,4 +34,5 @@
   // "USERPREFS_USE_ADMIN_KEY_0": "{ 0xcd, 0xc0, 0xb4, 0x3c, 0x53, 0x24, 0xdf, 0x13, 0xca, 0x5a, 0xa6, 0x0c, 0x0d, 0xec, 0x85, 0x5a, 0x4c, 0xf6, 0x1a, 0x96, 0x04, 0x1a, 0x3e, 0xfc, 0xbb, 0x8e, 0x33, 0x71, 0xe5, 0xfc, 0xff, 0x3c }",
   // "USERPREFS_USE_ADMIN_KEY_1": "{}",
   // "USERPREFS_USE_ADMIN_KEY_2": "{}"
+  // "USERPREFS_CONFIG_DISCOVERY_CHANNEL_HASH": "8"
```

Comment:
Let's change this to something like `USERPREFS_NODEINFO_BROADCAST_CHANNEL_HASH`, since it's not a normal config option.